styles/index.scss

```diff
@@ -15,13 +15,29 @@ $heading-font: 'Nunito', 'Avenir', sans-serif;
   width: 100%;
 }
 
-.section{
+.top-photo {
+  width: 575px;
+  height: 530px;
+  overflow: hidden;
```

Comment:
this overflow:hidden seems to give the appearance of the image getting covered on screens that are small but not small enough to switch to the mobile layout

Reply:
Is there something in particular you want for this instead?

Reply:
the main image (the one that used to have overflow) looks fine now and correctly scales with the page so it shouldn't need anything to replace the overflow